mti_nma/steps/avgshape/avgshape.py

```diff
@@ -71,6 +71,9 @@ def run(self, **kwargs):
             filename=str(avg_data_dir / Path('avgshape.vtk'))
         )
 
+        # Save mesh as stl file for blender import
+        save_mesh_as_stl(mesh_avg, str(avg_data_dir / Path('avgshape.stl')))
```

Comment:
Don't need to convert that to a path first. You can do `{path} / {str}` just fine.